Lib/test/test_dis.py

```diff
@@ -1050,12 +1105,12 @@ def test_explicit_first_line(self):
         self.assertEqual(list(actual), expected_opinfo_outer)
 
     def test_source_line_in_disassembly(self):
-        # Use the line in the source code
-        actual = dis.Bytecode(simple).dis()[:3]
+        # Use the line in the source code (split extracts the line no)
+        actual = dis.Bytecode(simple).dis().split(" ")[0]
```

Comment:
To my eye, it's clearer this way, since the focus is on what we're doing to the dis line and the verbs are all simple: strip whitespace from the left, split on whitespace and pick the first region. 

The main clarity improvement would seem to be refactoring out an `extract_line_num` method, but per my commit comment that seems like overkill in test code, and I'm not sure what the best place to put it would be.

Others?